client/src/HomeNavbar.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box, Typography } from '@mui/material';
+
+import { Link } from 'react-router-dom';
+import React from 'react';
+
+const HomeNavbar = () => {
+  return (
+    <Box sx={{ position: 'absolute' }}>
```

Comment:
Slight suggestion, nothing necessary. Next time try to write it as shown below, it's dedicated way to do it by MUI and we should follow it.
```suggestion
    <Box position="absolute">
```